### DISCUSSION TITLE
Weird Fix for "Error: Could not create Sunshine Mouse: Permission denied"

### DISCUSSION THREAD

**Discussion by jdancouga** [LizardByte/community#22] [Sunshine] [answered]
I was trying out Linux Mint (20.3) and used the AppImage version of sunshine, which works straight away. Only thing I needed to do was adding the user to group: input. However, later I messed up my OS when trying to install Battle.net/Diablo2R through Lutris. I decided to just wipe the drive and reinstall the entire OS. Somehow after doing so, I kept getting the error in the title from the sunshine AppImage. I did exactly what I did the first time but it just wouldn't work.

After searching the web, I found these two similar issues [1](https://github.com/loki-47-6F-64/sunshine/issues/52) & [2](https://github.com/loki-47-6F-64/sunshine/issues/147), but the solutions didn't work for me.

In desperation, I tried the instructions from the Building section of Readme:
**Run the following command: nano /etc/udev/rules.d/85-sunshine-input.rules
Input the following contents: KERNEL=="uinput", GROUP="input", MODE="0660"**
This somehow fix the problem for me. Not sure why, but just want to put this on the web in case someone has similar issue.

**Comment by ReenigneArcher** [comment, marked as answer]
Those aren't the build instructions, they're the instructions needed to use Sunshine. The readme is kind of confusing. I've tried to make it more clear in the wiki, here: https://github.com/SunshineStream/Sunshine/wiki/Usage#linux

**Reply by jdancouga** [reply]
Thanks for the clarification. I didn't realize this is part of the setup process. When I tried Manjaro, I just had to one click install from AUR and didn't have to do this. I made a wrongfully assumption. Now reading through the code carefully, AUR package did this automatically as part of the installation process.

My noob-ness got me after all.
